server/handler/base.go

```diff
@@ -193,5 +206,58 @@ func (b *Base) EvaluateFetchedConfig(ctx context.Context, prctx pull.Context, cl
 		return errors.Errorf("evaluation resulted in unexpected state: %s", result.Status)
 	}
 
+	if statusState == "pending" && prctx.IsDraft() != nil && !*prctx.IsDraft() {
+		subsetCurrentReviewers, _, err := client.PullRequests.ListReviewers(ctx, prctx.RepositoryOwner(), prctx.RepositoryName(), prctx.Number(), &github.ListOptions{
+			Page:    0,
+			PerPage: 10,
+		})
+
+		if err != nil {
+			logger.Warn().Err(err).Msg("Unable to list request reviewers")
+		}
+
+		if subsetCurrentReviewers != nil && len(subsetCurrentReviewers.Users) == 0 && len(subsetCurrentReviewers.Teams) == 0 {
+			// look for requested reviewers
+			leafNodes := findLeafChildren(result)
+			var requestedUsers, requestedTeams []string
+			for _, child := range leafNodes {
+				if child.Error != nil {
+					continue
+				}
+
+				requestedUsers = append(requestedUsers, child.RequestedUsers...)
+				requestedTeams = append(requestedTeams, child.RequestedTeams...)
+			}
+
+			// remove PR author if they exist in the possible list, since they cannot be assigned to their own PRs
+			for i, u := range requestedUsers {
+				if u == prctx.Author() {
+					requestedUsers = append(requestedUsers[:i], requestedUsers[i+1:]...)
+				}
+			}
+
+			// remove the owner org name from teams, if present
+			// https://developer.github.com/v3/pulls/review_requests/#create-a-review-request
+			for i, t := range requestedTeams {
+				if strings.Contains(t, prctx.RepositoryOwner()) {
+					requestedTeams[i] = strings.ReplaceAll(t, fmt.Sprintf("%s/", prctx.RepositoryOwner()), "")
```

Comment:
I think we'll also need to discard teams that are _not_ part of the organization, so once we do that, it should be fine to split these on first slash, without regard to the prefix.

If you do use the prefix, `strings.TrimPrefix(...)` would be clearer than `strings.ReplaceAll(...)`.